documents-webapp/src/main/webapp/vue-app/documents/components/body/table/DocumentInfoDrawer.vue

```diff
@@ -209,7 +209,9 @@ export default {
       return this.$vuetify.breakpoint.name === 'xs' || this.$vuetify.breakpoint.name === 'sm';
     },
     disableButton() {
-      return !this.file || !this.file.description || this.file.description.replace( /(<([^>]+)>)/ig, '').length>1300;
+      return this.file.description && this.file.description.replace( /(<([^>]+)>)/ig, '').length>1300
+      || this.file.description === this.fileInitialDescription 
+      || this.file.description === '' && this.fileInitialDescription === null;
```

Comment:
Is this correct too ?
```suggestion
      || (!this.file.description && !this.fileInitialDescription);
```